src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,27 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => (
+  <>
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {' Posted by  '}
```

Comment:
you can wrap with `span`
```suggestion
          <span> Posted by  </span>
```